src/content/docs/en/reference/api-reference.mdx

```diff
@@ -874,16 +874,21 @@ The Markdown component is no longer built into Astro. See how to [import Markdow
 
 ### `<Code />`
 
-```astro
+```astro 'theme="dark-plus"' /wrap\b/ /(inline) \/>/
 ---
 import { Code } from 'astro/components';
 ---
 <!-- Syntax highlight some JavaScript code. -->
 <Code code={`const foo = 'bar';`} lang="js" />
-<!-- Optional: customize your theme. -->
+<!-- Optional: Customize your theme. -->
 <Code code={`const foo = 'bar';`} lang="js" theme="dark-plus" />
 <!-- Optional: Enable word wrapping. -->
 <Code code={`const foo = 'bar';`} lang="js" wrap />
+<!-- Optional: Output inline code. -->
+<p>
+  <Code code={`const foo = 'bar';`} lang="js" inline />
+  will be rendered inline.
+</p>
```

Comment:
```suggestion
<p> Render <Code code={`const foo = 'bar';`} lang="js" inline /> inline.</p>
```
I kind of want to push back here.  :sweat_smile: I think the real reason to show any kind of example is to see it actually inline inside a sentence. It doesn't *look* obviously inline in the example above (even though it is).

I would suggest something like above, or if we *really* value consistency of visuals/formatting above all, then go back to the original `<Code code={`const foo = 'bar';`} lang="js" inline />` since this is a reference page and it can be more condensed.

Reply:
I think the `<p>` tag already signals that it's an inline content, so I'm not sure if we need to wrap with words to further emphasize it. Otherwise I'd also opt for the original example for consistency, but I do like the `<p>` tag there as an example. But these are also only my thoughts though.

Reply:
Good enough! Everyone else LGTM'd this, so let's release it as is!